brave/browser-laptop
# Refactor downloads bar with Aphrodite & Add toolbar submenu

**This blocks #10951**

- Create `Toolbar` submenu under `View` (to close #10548)

  <img width="431" alt="screenshot 2017-10-16 14 53 45" src="https://user-images.githubusercontent.com/3362943/31597158-dd364ff2-b281-11e7-8b79-9da3573be9f4.png">

  TODO for a follow-up PR: Add `Tabs Toolbar` and `Navigation Bar` to hide everything except the menu bar on Windows

Fixes #9320
Closes #10548
Closes #10946 
Closes #10948
Addresses #10947 



Test Plan:
`npm run unittest -- --grep='downloadsToolbarMenuItem'`
`npm run test -- --grep='Downloads'`

Test Plan 2:
1. Click a `100MB` file from `http://speedtest.reliableservers.com/`
2. Make sure the `Close` button is displayed on the download bar
3. Hover on the file item
4. Make sure the action buttons are displayed
5. Click the pause icon
6. Make sure the transition animation is canceled
7. Click the trash icon
8. Make sure the delete confirmation button works

Test Plan 3:
1. Hide `Download Bar` from `Menu`
2. Download something
3. Make sure the download bar is hidden

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
